pkg/controllers/management/k3supgrade/deployPlans.go

```diff
@@ -156,6 +156,13 @@ func (h *handler) modifyClusterCondition(cluster *v3.Cluster, masterPlan, worker
 	// implement a simple state machine
 	// NotUpgraded => MasterPlanUpgrading || WorkerPlanUpgrading =>  NotUpgraded
 
+	if masterPlan.Name == "" && workerPlan.Name == "" {
+		v3.ClusterConditionUpgraded.Unknown(cluster)
+		genericUpgradeMessage := fmt.Sprintf("cluster is being upgraded")
```

Comment:
nit: we don't need this var and can directly pass the string to `v3.ClusterConditionUpgraded.Message`? rest LGTM

Reply:
Sounds like a good change